CONTRIBUTING.md

```diff
@@ -1,34 +1,115 @@
-# Contributing to scrumlr.io
+# Contributing
+Thank you for considering contributing to Scrumlr and taking the time to explore our contribution guide. We greatly appreciate your interest and willingness to help improve Scrumlr with your valuable contributions. Your involvement and contributions play a significant role in enhancing Scrumlr and making it even better. We warmly welcome pull requests from all contributors, big or small. For major changes, we kindly request to [open a discussion](https://github.com/inovex/scrumlr.io/discussions) first, allowing us to collaborate and discuss your ideas and proposed changes. Once again, we extend our heartfelt gratitude for your dedication and passion in helping us shape the future of Scrumlr.
```

Comment:
I'd cut the last sentence, it seems too much imo

Reply:
I think it's actually pretty good in expressing our gratitude towards the people willing to contribute and improve scrumlr :)

Reply:
sure why not 👍